### PR TITLE
async sql paging updates

### PR DESCRIPTION
**Pull request by sergioferragut**
-Updated druidapi package async_sql function to allow for pagination
- Updated docker compose and environment to configure MSQ using deep storage which is required for pagination
- Added the pagination example to the async notebook

### CODE REVIEW

File: jupyter-img/druidapi/druidapi/sql.py [hunk 1]
```diff
@@ -809,6 +809,19 @@ def rows(self):
                     page+=1
         return self._rows
```
Comment (petermarshallio) [inline jupyter-img/druidapi/druidapi/sql.py:810]: A note for the line at the start re: the version of Druid required for this notebook (it says 28).

Also, that the old notebook for 28 should be retained with a suffix of "-28" as per the lookups one.

File: notebooks/03-query/14-sync-async-queries.ipynb [hunk 2]
```diff
@@ -600,6 +600,49 @@
...
   },
+  {
+   "cell_type": "markdown",
+   "id": "21a32041-e9cb-4b14-a149-b63ccd4c0386",
+   "metadata": {},
+   "source": [
+    "# Retrieving results page by page \n",
+    "The API allows the user to retrieve results a page at a time.\n",
+    "The size of each page can be controlled with the query context parameter `rowsPerPage`.\n",
+    "The `druidapi` package now incorporates the rowsPerPage parameter into the Druid API call.  \n",
```
Comment (petermarshallio) [inline notebooks/03-query/14-sync-async-queries.ipynb:611]: Suggest rephrase as

"Use the `rowsPerPage` query context parameter to control the size of the results page.

In the cell below, the page size is set by providing a parameter in the call to the Python API."

**Review by petermarshallio — CHANGES_REQUESTED** [review]
Unfortunately I couldn't complete the review as I got :

QueryClient.async_sql() got an unexpected keyword argument 'rowsPerPage'

I'm presuming I'm not using the right Python API version?

Else it's just a few phrasing things.

File: jupyter-img/druidapi/druidapi/sql.py (hunk 1 above)
Comment (sergioferragut) [reply jupyter-img/druidapi/druidapi/sql.py:810]: Let's talk about this... I don't think we want this growing list of versioned notebooks.
Regarding it not working for you, it seems like you were using the old jupyter image. How did you start the docker compose?

File: notebooks/03-query/14-sync-async-queries.ipynb (hunk 2 above)
Comment (sergioferragut) [reply notebooks/03-query/14-sync-async-queries.ipynb:611]: done

File: jupyter-img/druidapi/druidapi/sql.py (hunk 1 above)
Comment (sergioferragut) [reply jupyter-img/druidapi/druidapi/sql.py:810]: Applied new versioning strategy including link in README

**Pull request merged**